application-flashmessages-ui/src/main/resources/Flash/DisplayFlashMessages.xml

```diff
@@ -275,7 +275,7 @@
 #flashMessages {
   background-color: $theme.menuBackgroundColor;
   box-shadow: 0 2px 8px rgba(0, 0, 0, 0.4) inset;
-  color: #fff;
+  color: $theme.menuLinkColor;
```

Comment:
The top slider color's change comes in the new XWiki versions. I propose for this version to change just the color of the text and to use a variable which fits to the existing slider color (i.e $theme.menuBackgroundColor with $theme.menuLinkColor)